src/components/Filter/Filter.jsx

```diff
@@ -0,0 +1,78 @@
+import "./Filter.css";
+
+const Filter = () => {
+  return (
+    <aside className="filter-container">
+      <div className="filter-heading">
+        <h2 className="main-heading">Filters</h2>
+        <button className="filter-clear-btn">Clear</button>
+      </div>
+
+      <div className="filter-type">
+        <h3 className="filter-type-heading">Price</h3>
+        <div className="filter-type-container">
+          <label for="cost-slider">Rs.0 - Rs.1000</label>
+          <input type="range" id="cost-slider" min="0" max="1000" value="500" />
+        </div>
+      </div>
+
+      <div className="filter-type">
+        <h3 className="filter-type-heading">Category</h3>
+        <div className="filter-type-container">
+          <input type="checkbox" id="healthy" />
+          <label for="healthy">Healthy</label>
```

Comment:
You can use **htmlFor** attribute instead of for

Reply:
Didn't understand the reason at first but after doing a bit of reading, I understood why `for` attribute needed to be changed to `htmlFor`. Thanks for helping me learn something new 😄